Sources/GitHub/Client.swift

```diff
@@ -36,9 +35,11 @@ extension Client {
 
   public init(clientId: Id, clientSecret: Secret, logger: Logger?) {
     self.init(
-      fetchAuthToken: fetchGitHubAuthToken(clientId: clientId, clientSecret: clientSecret) >>> runGitHub(logger),
-      fetchEmails: fetchGitHubEmails >>> runGitHub(logger),
-      fetchUser: fetchGitHubUser >>> runGitHub(logger)
```

Comment:
removing dependence on `>>>`, we could prob uncurry `runGitHub` and use trailing closures to make this a lil nicer